setup/tue-functions.bash

```diff
@@ -19,66 +19,6 @@ function _list_subdirs
     done
 }
 
-function _set_export_option {
-
-    # _set_export_option KEY VALUE FILE
-    # Add the following line: 'export KEY=VALUE' to FILE
-    # Or changes the VALUE to current value if line already in FILE.
-
-    key=${1//\//\\/}
-    value=${2//\//\\/}
-    sed -i \
-        -e '/^#\?\(\s*'"export ${key}"'\s*=\s*\).*/{s//\1'"${value}"'/;:a;n;ba;q}' \
-        -e '$a'"export ${key}"'='"${value}" $3
-}
-
-# ----------------------------------------------------------------------------------------------------
-#                                              SSH
-# ----------------------------------------------------------------------------------------------------
-
-function tue-use-ssh
-{
-    local option="TUE_USE_SSH"
-    local value="true"
-    _set_export_option "$option" "$value" $TUE_ENV_DIR/.env/setup/user_setup.bash
-    source $TUE_ENV_DIR/.env/setup/user_setup.bash
-}
-
-function tue-use-https
-{
-    local option="TUE_USE_SSH"
-    local value="false"
-    _set_export_option "$option" "$value" $TUE_ENV_DIR/.env/setup/user_setup.bash
-    source $TUE_ENV_DIR/.env/setup/user_setup.bash
-}
-
-function _github_https
```

Comment:
All the github functions are needed in the installer. They are also exported so they are available in tue-env config.

Reply:
Expand the file and below the removed area and you will see they are there. They ended up being defined twice due to the merge